package-lock.json

```diff
@@ -3067,13 +3042,15 @@
         },
         "isarray": {
           "version": "1.0.0",
-          "bundled": true,
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
```

Comment:
I don't fully understand this part but was there a reason of moving to resolved?
Also, isn't this file auto generated 🤔

Reply:
package-lock.json is autogenerated, when package.json is changed we delete the lock file, run npm step to re-gen the lock file, and commit the lock file so that the package versions are pinned. I don't pay close attention to the contents of the lock file so I am not sure about your question.